src/app/tests/suites/TestFanControl.yaml

```diff
@@ -69,6 +69,8 @@ tests:
       attribute: "PercentSetting"
       arguments:
           value: null
+      response:
+          error: INVALID_IN_STATE
```

Comment:
We need to add a cert test for this in TC-FAN-XXXX. Please open an issue to update a test script plan and create SDK issue to update test.

Reply:
This is already covered in the fan control cert test python scripts, @yinyihu-silabs confirmed it.

Reply:
[TC_FAN_3_1.py](https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_FAN_3_1.py)
I understand there is plan to update fan cluster test plan.